chemprop/train/loss_functions.py

```diff
@@ -148,7 +148,8 @@ def mcc_multiclass_loss(
     if cov_ypyp * cov_ytyt == 0:
         loss = torch.tensor(0.0)
```

Comment:
Should this line also be subtracted from 1?

Reply:
That loss is undefined, presumably due to bad coverage of the confusion matrix by the minibatch. I set it as a fixed zero so that one bad batch wouldn't break the model. It's already detached from the gradients so it won't matter for descent whether it's 1 or 0. But I suppose for training loss tracking it would be better for it to reflect a bad model. Will make that change along with fixing the testing values.